docs/pages/tutorial/eas/android-development-build.mdx

```diff
@@ -0,0 +1,191 @@
+
+To create a **.apk**, make sure:
+
+- The `developmentClient` is set to `true` in **eas.json**, which is the default configuration.
+- Then, run the `eas build` command with `android` as the platform and `development` as the build profile:
+
+<Terminal cmd={['$ eas build --platform android --profile development']} />
+
+> **info** **Tip**: Next time you run `eas build` command, you can also use `-p` to specify the platform. It is short for `--platform`.
+
+This command prompts us with the following questions:
+
+- **What would you like your Android application id to be?** Press <kbd>return</kbd> to select the default value provided for this prompt.
+- **Generate a new Android Keystore**? Press <kbd>Y</kbd>.
+
+<Collapsible summary="What is an Android application ID?">
+
+Also known as the package name of our Android app, it stores the value in DNS reverse notation format (`com.owner.app-name`). Each component of this notation should start with a lowercase letter.
+
+For example, our example app has `com.owner-name.stickersmash` where `com.owner-name` is the domain and `stickermsash` is our app name.
+
+</Collapsible>
+
+When we answer these prompts, the EAS CLI updates the [`android.package`](/versions/latest/config/app/#package) property in **app.json**. After responding to the prompts, our EAS Build is queued, and the EAS CLI provides a link to view build details and track progress on the Expo dashboard:
+
+<ImageSpotlight
+  alt="Android development build details and progress in Expo dashboard"
+  src="/static/images/tutorial/eas/android-build-details.png"
+  style={{ maxWidth: 800 }}
+/>
+
+<Collapsible summary="What information does a build details page contain?">
+
+The build details page displays the build type, profile, Expo SDK version, app version, version code, last commit hash, and the identity of the developer or account owner who initiated the build.
+
+In the above image, the current status of the **Build artifact** shows that the build is in progress. Upon completion, this section will offer an option to download the build. The **Logs** outlines every step taken during the Android build process on EAS Build. For the sake of brevity, we won't explore each step in detail here. To learn more, see [Android build process](/build-reference/android-builds/).
+
+</Collapsible>
+
+</Step>
+
+<Step label="2">
+
+## Install development build on Android device
+
+Once the build finishes, the **Build artifact** section gets updated, indicating that the build is complete:
+
+<ImageSpotlight
+  alt="Build artifact gives the option to download development build for Android devices and emulators"
+  src="/static/images/tutorial/eas/android-build-artifact.png"
+/>
+
+This section provides the methods available for running the development build on an Android device:
+
+- Open with Expo Orbit (macOS only)
```

Comment:
Windows and Linux support [coming soon](https://linear.app/expo/project/orbit-v200-support-on-windows-and-linux-03a10a450949) 😄

Reply:
When that lands, we'll update this tutorial and hopefully make the next step as an alternate.

Reply:
I think I'm going to remove "macOS only" part as I heard from Brent the support for Windows is coming soon.